src/amp-story-player/amp-story-player-impl.js

```diff
@@ -434,6 +436,21 @@ export class AmpStoryPlayer {
     });
   }
 
+  /**
+   * Gets publisher configuration for a custom system UI.
+   * @private
+   * @return {?JsonObject}
+   */
+  getCustomUI_() {
+    if (childElementsByTag(this.element_, 'script').length === 0) {
+      return null;
+    }
+
+    const config = getChildJsonConfig(this.element_);
```

Comment:
These two helpers bring in quite a lot of AMP helpers into the build. Can we do it with simple JavaScript to save some precious bytes? It should be pretty straightforward.

Reply:
Good call, decided to just keep `isJsonScriptTag()` since it doesn't bring any other helpers and I don't expect it to change in the future.